app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
+
+
+class NotWearingMaskError(Exception):

```

Comment:
The `NotWearingMaskError` class should inherit from `VaccineError` instead of `Exception` to maintain consistency with the other error classes and ensure proper exception handling.